moto/timestreaminfluxdb/models.py

```diff
@@ -270,6 +286,255 @@ def untag_resource(self, resource_arn: str, tag_keys: List[str]) -> None:
     def list_tags_for_resource(self, resource_arn: str) -> Dict[str, str]:
         return self.tagger.get_tag_dict_for_resource(resource_arn)
 
+    def create_db_parameter_group(
+        self,
+        name: str,
+        description: Optional[str] = None,
+        parameters: Optional[Dict[str, Any]] = None,
+        tags: Optional[Dict[str, str]] = None,
+    ) -> Tuple[str, str, str, str, Dict[str, Any]]:
+        validate_name(name)
+
+        for param_group in getattr(self, "db_parameter_groups", {}).values():
+            if param_group["name"] == name:
+                raise ConflictException(
+                    f"A DB parameter group with the name {name} already exists"
+                )
+
+        if not hasattr(self, "db_parameter_groups"):
+            self.db_parameter_groups = {}
+
+        param_group_id = random_id()
+
+        arn = f"arn:aws:timestream-influxdb:{self.region_name}:{self.account_id}:db-parameter-group/{param_group_id}"
+
+        param_group = {
```

Comment:
Can we make this a separate class, `ParameterGroup(BaseModel)`, instead of dealing with a dictionary? That makes it a bit easier to reason over.